.ci_support/osx_64_build_typereleasechannel_targetsconda-forge_python_debug.yaml

```diff
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 MACOSX_SDK_VERSION:
 - '11.0'
 build_type:
-- debug
+- release
```

Comment:
That ain't right...
Somehow the wrong values seem to have been zipped together.
I haven't yet checked whether I can reproduce this locally...
cc @conda-forge/core for awareness

Reply:
Can't reproduce for now.
Welp, an adventure for another day, I guess...